srtcore/core.cpp

```diff
@@ -8795,6 +9527,10 @@ int CUDT::processData(CUnit* in_unit)
             // Otherwise it's an error.
             if (adding_successful)
             {
+                // XXX move this code do CUDT::defaultPacketArrival and call it from here:
+
+                // srt_loss_seqs = CALLBACK_CALL(m_cbPacketArrival, rpkt);
+
```

Comment:
Not sure whether this comment should stay, but at least no need in extra blank lines

Reply:
Should stay. This is where the parallel packet lossreport prevention should be added. It's an important improvement for broadcast groups.